controllers/propagator/propagation.go

```diff
@@ -544,30 +576,45 @@ func (r *PolicyReconciler) handleRootPolicy(instance *policiesv1.Policy) error {
 	if err != nil {
 		reqLogger.Error(err, "Giving up on updating the root policy status...")
 		r.recordWarning(instance, "Failed to update the policy status")
+
 		return err
 	}
 
 	err = r.cleanUpOrphanedRplPolicies(instance, allDecisions)
 	if err != nil {
 		reqLogger.Error(err, "Giving up on deleting the orphaned replicated policies...")
 		r.recordWarning(instance, "Failed to delete orphaned replicated policies")
+
 		return err
 	}
 
 	reqLogger.Info("Reconciliation complete.")
+
 	return nil
 }
 
 // getApplicationPlacementDecisions return the placement decisions from an application
 // lifecycle placementrule
-func getApplicationPlacementDecisions(c client.Client, pb policiesv1.PlacementBinding, instance *policiesv1.Policy) ([]appsv1.PlacementDecision, *policiesv1.Placement, error) {
+func getApplicationPlacementDecisions(
+	c client.Client, pb policiesv1.PlacementBinding, instance *policiesv1.Policy,
+) ([]appsv1.PlacementDecision, *policiesv1.Placement, error) {
 	plr := &appsv1.PlacementRule{}
-	err := c.Get(context.TODO(), types.NamespacedName{Namespace: instance.GetNamespace(),
-		Name: pb.PlacementRef.Name}, plr)
+
+	err := c.Get(context.TODO(), types.NamespacedName{
+		Namespace: instance.GetNamespace(),
+		Name:      pb.PlacementRef.Name,
+	}, plr)
 	// no error when not found
 	if err != nil && !k8serrors.IsNotFound(err) {
-		log.Error(err, "Failed to get PlacementRule...", "Namespace", instance.GetNamespace(), "Name",
-			pb.PlacementRef.Name)
+		log.Error(
+			err,
+			"Failed to get PlacementRule...",
+			"Namespace",
+			instance.GetNamespace(),
+			"Name",
+			pb.PlacementRef.Name,
+		)
```

Comment:
Again, I would prefer structured logging keys and values on the same line, if not prohibited by a linter.